cf-231A.mdx

```diff
@@ -0,0 +1,154 @@
+---
+id: cf-231A
+source: CF
+title: Team
+author: Sathvik Chundru
+---
+
+[Official Analysis (CPP)](https://codeforces.com/blog/entry/5486)
+
+## Solution (Greedy)
+
+Let's consider the following test case:
+
+```
+3
+1 1 0
+1 1 1
+1 0 0
+```
+
+There are 2 ways to solve this problem. One solution is to input all the ones
+and zeros into a two-d array and sort through them manually to see how many
+cases have two or more ones. The second simpler solution is to input three
+integers per row and check if their sum is greater than or equal to two. Both
+solutions are implemented below.
+
+## Implementation
+
+<LanguageSection>
+
+<CPPSection>
+
+```CPP
+#include <iostream>
+using namespace std;
+
+int main() {
+	int num = 0;
+	int n;
+	cin >> n;
+	int problems[n][3];
+	for (int i = 0; i < n; i++) {
+		cin >> problems[i][0];
+		cin >> problems[i][1];
+		cin >> problems[i][2];
+	}
+	for (int i = 0; i < n; i++) {
+		int temp = 0;
+		for (int j = 0; j < 3; j++) {
+			if (problems[i][j] == 1) {
+				temp++;
+			}
+		}
+		if (temp >= 2) {
+			num++;
+		}
+	}
+	cout << num << '\n';
+}
+```
+
+</CPPSection>
+
+<CPPSection>
+
```

Comment:
why are there two cpp sections adjacent to each other

Reply:
cuz harry told me to 😭

Reply:
i told u to language section and move the respective editorials to the sections 👀